test/camunda-cloud/ElementTemplatesReplaceProviderSpec.js

```diff
@@ -323,6 +326,63 @@ describe('<ElementTemplatesReplaceProvider>', function() {
 
   });
 
+
+  describe('integration with connectors extension', function() {
+
+    beforeEach(bootstrapCamundaCloudModeler(
+      diagramXML, {
+        additionalModules: [
+          ConnectorsExtensionModule
+        ],
+        connectorsExtension: {
+          appendAnything: true
+        },
+      }
+    ));
+
+    beforeEach(inject(function(elementTemplates) {
+      elementTemplates.set(templates);
+    }));
+
+
+    it('templates are not duplicated', inject(function(elementRegistry, selection, replaceMenu, changeMenu) {
```

Comment:
You could make this part of the cloud [`ModelerSpec`](https://github.com/camunda/camunda-bpmn-js/blob/main/test/camunda-cloud/ModelerSpec.js). I'd suggest to create a `start:xxx` script, too so the integration can be tested easily manually (cf. [existing](https://github.com/camunda/camunda-bpmn-js/blob/main/test/camunda-cloud/ModelerSpec.js#L137) cloud version).